test/ActiveLogin.Authentication.BankId.AspNetCore.Test/BankId_Authentication_Tests.cs

```diff
@@ -164,16 +164,18 @@ public async Task BankIdAuthentication_Login_Returns_Form_And_Status()
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, transaction.StatusCode);
+
             var content = await transaction.Content.ReadAsStringAsync();
             Assert.Contains("<form id=\"bankIdLoginForm\">", content);
-            Assert.Contains("<div id=\"bankIdLoginStatus\"", content);
+            Assert.Contains("<div class=\"card\" id=\"bankIdLoginStatus\" style=\"display: block;\">", content);
```

Comment:
Don't need to change this now, but I recently stumbled upon [AngleSharp ](https://anglesharp.github.io/) for parsing HTML in C#. Microsoft seems to recommend it here: https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.0

If I get it right, you can parse the DOM using the lib, query for the element you like and check a specific prop. That would make our tests less fragile.

Reply:
Very nice! I created an issue for it #147